File: 1 - Beginner/1010 Simple Calculate/Python/main.py
cod1, quant1, price1 = input("").split()
cod2, quant2, price2 = input("").split()
total = int(quant1)*float(price1)
total += int(quant2)*float(price2)
print("VALOR A PAGAR: R$ "+str(round(total,2)))

File: 1 - Beginner/1010 Simple Calculate/C/Main.cpp
#include <stdio.h>

int main(void){
int a1,b1,a2,b2; // a => ID | b => Quantity  
float c1,c2,total; // Price
scanf("%d %d %f",&a1,&b1,&c1);
scanf("%d %d %f",&a2,&b2,&c2);
total = b1*c1;
total+= b2*c2;
printf("VALOR A PAGAR R$ %.2f",total);
}


